_posts/2016-8-28-report-world-moveit-day.markdown

```diff
@@ -0,0 +1,87 @@
+---
+author: davetcoleman
+comments: false
+date: 2016-08-28 08:15:12+00:00
+layout: post
+slug: wordlmoveitdayreport
+title: Report on World MoveIt! Day
+media_type: image
+media_link: http://moveit.ros.org/assets/images/report_on_world_moveit_day.jpg
+description: Results from our first international hackathon to improve the MoveIt! code base, documentation, and community.
+categories:
+- MoveIt!
+- ROS
+---
+
+![World MoveIt! Day]({{ site.url }}/assets/images/report_on_world_moveit_day.jpg)
+
+Thank you for joining us for our first international MoveIt! hackathon event! It was a great success with many participants from around the world joining, many issues fixed, and features added. At the Fetch event in California we had 18 participants join throughout the day.
```

Comment:
Could you add "[we had 18] and at the Xamla event in Germany 9 [participants]" to the sentence?